components/provider_cldr/src/transform/dates.rs

```diff
@@ -374,6 +451,26 @@ pub(self) mod cldr_json {
         pub short: StylePattern,
     }
 
+    #[derive(PartialEq, Debug, Deserialize)]
+    pub struct DateTimeFormats {
+        pub full: StylePattern,
+        pub long: StylePattern,
+        pub medium: StylePattern,
+        pub short: StylePattern,
+        #[serde(rename = "availableFormats")]
+        pub available_formats: AvailableFormats,
+    }
+
+    #[derive(PartialEq, Clone, Debug, Deserialize)]
+    pub struct AvailableFormats(
+        #[serde(with = "tuple_vec_map")] pub(crate) Vec<(Cow<'static, str>, Cow<'static, str>)>,
```

Comment:
Since the CLDR JSON is only ever created from `serde::from_reader`, these strings will never be static.  Therefore, I think we should start using plain `String` in CLDR JSON structs (only in CLDR JSON, not in ICU4X JSON!)

Reply:
I agree, however I would like this to be a follow-up, since this code is a change to the existing practice. I filed #558